pkg/lock/stoppable_waitgroup_test.go

```diff
@@ -135,7 +135,7 @@ func TestWaitChannel(t *testing.T) {
 	default:
 	}
 
-	l.Done()
+	done2()
```

Comment:
Nit. I'd call `done3()` before this one, and assert that the counter didn't change, to validate the behavior that is being fixed by this commit.